test/test_torch.py

```diff
@@ -2853,6 +2853,25 @@ def test_arange_inference(self):
                                    torch.tensor(1, dtype=torch.int16)).dtype)
         torch.set_default_dtype(saved_dtype)
 
+    def test_randint_inference(self):
+        saved_dtype = torch.get_default_dtype()
+        torch.set_default_dtype(torch.float32)
```

Comment:
I don't think you rely on this anywhere (the default dtype can't be an integral type).

Reply:
good point. i'll remove.